xbmc/pvr/windows/GUIWindowPVRSearch.cpp

```diff
@@ -22,6 +22,7 @@
 #include "dialogs/GUIDialogOK.h"
 #include "dialogs/GUIDialogProgress.h"
 #include "epg/EpgContainer.h"
+#include "GUIUserMessages.h" // GUI_MSG_UPDATE_SOURCES
```

Comment:
i thought this might be useful information, but if you don't like it i can remove it, np